sources/packages/backend/libs/integrations/src/services/sfas/sfas-restriction-import.service.ts

```diff
@@ -96,6 +128,36 @@ export class SFASRestrictionImportService
     }
   }
 
+  /**
+   * Creates notifications for newly created legacy student restrictions.
+   * @param entityManager entity manager to execute in transaction.
+   * @param referenceDate date used to filter the restrictions created after this date.
+   */
+  private async generateLegacyRestrictionNotifications(
+    entityManager: EntityManager,
+    referenceDate: Date,
+  ): Promise<void> {
+    const studentIDAlias = "studentId";
+    const studentsIDsWithNewLegacyRestrictions = await entityManager
+      .getRepository(StudentRestriction)
+      .createQueryBuilder("studentRestriction")
+      .distinct()
+      .select("studentRestriction.student.id", studentIDAlias)
+      .innerJoin("studentRestriction.restriction", "restriction")
+      .where("studentRestriction.createdAt >= :referenceDate", {
```

Comment:
Why could it not be `createdAt = :referenceDate`

Reply:
It can.

Reply:
Changed as suggested.